fakenet/listeners/ProxyListener.py

```diff
@@ -0,0 +1,397 @@
+import socket
+import SocketServer
+import threading
+import sys
+import glob
+import time
+import importlib
+import Queue
+import select
+import logging
+import ssl
+from OpenSSL import SSL
+from ssl_utils import ssl_detector 
+import hexdump
+import listeners
+from listeners import *
```

Comment:
Going forward I'd advise against wildcard imports. As a reviewer (and as the code base grows), wildcard imports make it hard to determine where code is coming from.